sentry_streams/tests/adapters/arroyo/test_consumer.py

```diff
@@ -228,7 +235,7 @@ def test_multiple_routes(
     consumer.add_step(
         MapStep(
             route=Route(source="source1", waypoints=[]),
-            pipeline_step=cast(Map, router_pipeline.steps["decoder"]),
+            pipeline_step=cast(Map, cast(ComplexStep, router_pipeline.steps["decoder"]).convert()),
```

Comment:
why is this double-cast necessary?

Reply:
Because `MapStep` expects specifically a `Map` step. The output of `convert` is a general `Step`.